3.10/release-notes-upgrading-changes310.md

```diff
@@ -199,3 +199,12 @@ Using the old option names (`--query` and `--query-max-runtime`) is still
 supported and will implicitly use the `--custom-query` and
 `--custom-query-max-runtime` options under the hood. Client scripts should
 eventually be updated to use the new option name, however.
+
+### ArangoDB Starter
```

Comment:
I think it's worth mentioning that now starter will exit with code `1` in case if it encountered error while starting. This might affect user scripts. (see APM-380 / GT-100 for details)

Reply:
Actually the improvements related to exit codes are covered in this section: https://deploy-preview-1095--zealous-morse-14392b.netlify.app/docs/3.10/release-notes-upgrading-changes310.html#handling-of-invalid-startup-options

Reply:
The link you sent describes changes in `arangod` binary. But starter (`arangodb`) had a bug (APM-380), where it always exited with no error (0) even if it could not start components or some invalid CLI options was supplied. Now the bug is fixed and I think it's worth mentioning on release notes.

`arangodb` does not return exit codes specified in the link above.

This is difference between `arangod` and `arangodb` binaries was always confusing to me, but we have to document all the differences, otherwise end-users will experience this confusion too :)

Reply:
Thanks a lot for clarifying this! In this case, we should definitely mention it in the release notes.